spec/services/idv/steps/in_person/state_id_step_spec.rb

```diff
@@ -67,14 +67,15 @@
         expect(pii_from_user[:last_name]).to eq last_name
         expect(pii_from_user[:dob]).to eq formatted_dob
         expect(pii_from_user[:identity_doc_address_state]).to eq identity_doc_address_state
-        expect(pii_from_user[:state_id_number]).to eq state_id_number
+        # param from form as id_number but is renamed to state_id_number on update
```

Comment:
Same as above:

Instead of adding a new test to check that the params get translated back upon submit/"update" in the controller, I realized updating this test already verifies that happens. Just added this comment to illustrate that it is being checked here already.